src/api/config/routes/webui.rb

```diff
@@ -59,6 +59,7 @@
   resources :package, only: [:index], controller: 'webui/package', constraints: cons
   controller 'webui/requests_listing' do
     get 'requests', action: :index, constraints: cons, as: 'requests'
+    get 'projects/:project_name/requests/beta', action: :index, constraints: cons, as: :project_requests_beta
```

Comment:
Why do you append `/beta`?

Reply:
Because `projects/:project_name/requests` is already in use by the datatable.

Reply:
If we don't refactor this now, we will have to do it during rollout. I would at least keep a note of this in a review card

Reply:
I believe there should be a note/card for removing existing code in backlog. I can add one just incase

Reply:
https://trello.com/c/teHaOYvE/41-remove-existing-code-and-url-before-rollout

Reply:
This might be worth tackling before rollout, as it is a simple refactoring so that we don't have paths colliding